css/team.css

```diff
@@ -0,0 +1,345 @@
+/*
+===================================
+===================================
+Our Team Styles
+===================================
+===================================
+*/
+
+/*
+===================================
+Member Card Component
+===================================
+*/
+/* member card style */
+.member {
+	background-color: var(--color-teal);
+	color: var(--text-color-light);
+	padding: var(--spacing-sm);
+	border-radius: var(--border-radius);
+	max-width: 654px;
+}
+/* img styling*/
+.member .img {
+	max-width: 240px;
+}
+.member img {
+	border-radius: var(--border-radius);
+}
+/* text */
+.member h2 {
+	font-size: 28px;
+	font-weight: 700;
+	line-height: 32px;
+}
+
+.member .role {
+	font-size: 24px;
+	font-weight: 500;
+	line-height: 28px;
+}
+.member p {
+	font-size: 16px;
+	font-weight: 400;
+	line-height: 20px;
+}
+
+/* member card layout */
+.member,
+.member .text,
+.member .bio,
+.member-cat {
+	display: flex;
+}
+
+/* similar gaps */
+.member,
+.member .text,
+.member-cat {
+	gap: var(--spacing-sm);
+}
+
```

Comment:
Remove border-bottom, but we should have some kind of hover state for these icons. 

<img width="193" alt="Screen Shot 2024-01-11 at 3 50 53 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/448283ee-78ca-4118-8e1a-18e13cd29018">

Reply:
Thank  you! I forgot to apply the global img-link class